content/applications/hr/appraisals/reporting.rst

```diff
@@ -0,0 +1,188 @@
+the :guilabel:`Search...` bar.
+
+Next, click :guilabel:`Add Custom Filter` in the :guilabel:`Filters` section, and a :guilabel:`Add
+Custom Filter` pop up window loads.
+
+Using the drop-down menu, select :guilabel:`Status` for the :guilabel:`ID` field, then select
+:guilabel:`Cancelled` for the second drop-down field. Click the :guilabel:`Add` button, and only
+appraisals that have been cancelled appear.
+
+.. image:: reporting/custom-filter.png
+   :align: center
+   :alt: The Custom Filter pop-up with the parameters set to only show cancelled appraisals.
+
+Group appraisals
+~~~~~~~~~~~~~~~~
+
+The :guilabel:`Appraisal Analysis` report is grouped by :guilabel:`Department`, by default. This
+grouping can be removed, and/or different groupings can be added.
+
+To view the available grouping options, click the :icon:`fa-caret-down` :guilabel:`(caret down)`
+icon in the :guilabel:`Search...` bar.
+
+Active groupings have a green :icon:`fa-check` :guilabel:`(checkmark)` next to them. Click on a
+grouping option to change its current status, either from active to inactive (or vice versa). Any
+configuration of groupings can be selected.
+
+The available default :guilabel:`Group By` options are:
+
+- :guilabel:`Employee`: groups all appraisals by employee.
+- :guilabel:`Department`: groups all appraisals by department.
+- :guilabel:`Status`: groups all appraisals by their status.
+
+Skills evolution
+----------------
+
+To access the *Skills Evolution* report, navigate to :menuselection:`Appraisals app --> Reporting
+--> Skills Evolution`. The :guilabel:`Appraisal Skills Report` page displays a report of all skills,
+grouped by employee.
+
+Skill levels are **only** updated after an appraisal is marked as done. Any skill level changes from
+any ongoing appraisals that have **not** been finalized are **not** included in this report.
+
+All the lines of the report are collapsed, by default. To view the details of a line, click anywhere
+on a line to expand the data.
+
+Each skill has the following information listed:
+
+- :guilabel:`Employee`: name of the employee.
+- :guilabel:`Skill Type`: the category the skill falls under.
+- :guilabel:`Skill`: the specific, individual skill.
+- :guilabel:`Previous Skill Level`: the level the employee had previously achieved for the skill.
+- :guilabel:`Previous Skill Progress`: the previous percentage of competency achieved for the skill
+  (based on the :guilabel:`Skill Level`).
+- :guilabel:`Current Skill Level`: the current level the employee has achieved for the skill.
+- :guilabel:`Current Skill Progress`: the current percentage of competency achieved for the skill.
+- :guilabel:`Justification`: any notes entered on the skill, explaining the progress.
+
+The color of the skill text indicates any changes from the previous appraisal. Skill levels that
+have increased since the last appraisal appear in green, skill levels that have not changed appear
+in black, and skills that have *regressed* appear in red.
```

Comment:
Suggestion to include the filter name with the corresponding evolution color
```suggestion
The color of the skill text indicates any changes from the previous appraisal. Skill levels that
have increased since the last appraisal appear in green as an *Improvement*, skill levels that have
not changed appear in black as *No Change*, and skills that have regressed appear in red as
*Regression*.
```